UKHomeOffice/cerberus-service
# COP-11620 - Custom filter component

## Description
This PR adds a custom filter component (without trying to implement the form-renderer) which appears & operates the exact same way as the current filter.

This PR attempts to solve a few issues:
1) The current filter flickers and somewhat looks like it renders one thing then replaces it on either filter application or clearing filter.
2) Using the form renderer version, works in the sense that when it is applied, and cleared, clicking on apply again reapplied the previous selection.
3) Updating the form renderer version in attempt to solve the bug on Step 2. Clearing the filters does not remove the data within the FormRenderer though the component is being re-rendered as the prop has changed.

## To Test
- Pull & run against dev
- Manipulate the filter and observe behaviour which should not show the three issues raised above.
